### ISSUE TITLE
Autogenerate a schedule given constraints

### ISSUE THREAD

**Issue by skiadas** [sdp-resources/MajorPlanner#20]
We need to assign points

**Comment by CelesteHackmann** [comment]
5

**Comment by Hayesb20** [comment]
10 - i feel like this would be very difficult to make work because the system would have to search through all of a given majors requirements, and then 1. find the majors classes and its cognates that are available to satisfy those requirements 2. insert them into the schedule in a way that they are organized into the correct sequence (i.e CS223 isn't being taken before CS220, and do not violate recommended constrains (maybe?)  3. (maybe?) find electives that will fit to complete the major.

**Comment by CelesteHackmann** [comment]
I agree now that it is definitely more than a 5 because of what Brian has said

**Comment by smithma21** [comment]
Sounds pretty difficult and beyond the scope of what we might be able to accomplish at this time. I'd assign it at least a 10 like Brian.

**Comment by marsht21** [comment]
I also agree
